CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -209,6 +209,7 @@
       </trans-unit>
       <trans-unit id="Apple Keynote" xml:space="preserve">
         <source>Apple Keynote</source>
+        <target state="translated">Apple Keynote</target>
```

Comment:
Is this the same in German? Or did you miss this one by accident?

Reply:
Left it as-is on purpose. The Apple Keynotes are known by this name, as in "Hast du die Apple Keynote gesehen?"

I would not expect anyone to say "Hast du die Apple-Präsentation gesehen"; especially not as a title for the thing. If we use the reference to the Apple Keynote, we best leave it as is, is my POV.

Reply:
👍🏻 good deal, thanks. I assumed this wasn't an oversight, but I just wanted to double-check :)

Reply:
But I may be wrong here... @technocidal, what would you do here?

Reply:
@DonSqueak I completely agree with your assessment. Keynote is well-known and pretty common even outside of tech circles. Lot's of German conferences use it to advertise their opening talk for example.